oracle/src/sponge.rs

```diff
@@ -156,6 +156,10 @@ where
         }
     }
 
+    fn absorb_fq(&mut self, x: &[P::BaseField]) {
+        self.sponge.absorb(x)
```

Comment:
I don't remember too much since I wrote this a while ago, but in my version (simply called `absorb`) since it's implemented on the FqSponge, it resets `last_squeezed` first: https://github.com/o1-labs/proof-systems/pull/609/files#diff-aa0e65214a7b00c016dc4f8f67fdeae2dbfd7b77e2c18d01e74994f92fbe3efdR145